swarm.go

```diff
@@ -226,9 +226,8 @@ func NewSwarm(config *api.Config, mockStore *mock.NodeStore) (self *Swarm, err e
 		feedsHandler,
 	)
 
-	nodeID := config.Enode.ID()
-	self.netStore = storage.NewNetStore(lstore, bzzconfig.OverlayAddr, nodeID)
-	self.retrieval = retrieval.New(to, self.netStore, bzzconfig.OverlayAddr, self.swap) // nodeID.Bytes())
+	self.netStore = storage.NewNetStore(lstore, bzzconfig.Address)
+	self.retrieval = retrieval.New(to, self.netStore, bzzconfig.Address, self.swap) // nodeID.Bytes())
```

Comment:
Remove commented code.

Reply:
Ok, it was there before, so I did not erase it yet.

Reply:
Yes, I did not notice it, thanks.